app/views/katello/api/v2/repositories/show.json.rabl

```diff
@@ -60,8 +60,8 @@ node(:ssl_client_key, :unless => lambda { |repository| repository.ssl_client_key
   {:id => repository.ssl_client_key.id, :name => repository.ssl_client_key.name}
 end
 
-node :upstream_password_exists do |repo|
-  repo.upstream_password.present?
+node :upstream_auth_exists do |repo|
+  repo.upstream_username.present? && repo.upstream_password.present?
 end
```

Comment:
as much as i hate to, we shouldn't change the api here.  We can add to it, but we should leave the old value there as well